proto/qdrant/cloud/iam/v1/iam.proto

```diff
@@ -175,6 +175,19 @@ service IAMService {
       body: "*"
     };
   }
+  // Logs out the authenticated user.
+  // Required permissions:
+  // - None (authenticated only)
+  rpc Logout(LogoutRequest) returns (LogoutResponse) {
```

Comment:
we currently have `GetAuthenticatedUser` and `UpdateUser` methods, which contain the word `User` in the name. I'm wondering if we should do the same here and call it `LogoutUser`.

Reply:
I thought it would be redundant as there not be anything else to logout but there is no problem adding it of course.